pynest/examples/rate_neuron_DM.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would go for lowercase `_dm` in the file name.

Reply:
I agree, looks better with lowercase